html/supertokens_python/recipe/passwordless/emaildelivery/services/smtp/pless_login_email.html

```diff
@@ -511,13 +511,6 @@ <h1 class="title">Module <code>supertokens_python.recipe.passwordless.emaildeliv
                         text-decoration: underline;
                 }
 
-                @media only screen and (min-width:768px) {
-                        .templateContainer {
-                                width: 600px !important;
-                        }
-
```

Comment:
why is there a duplicate of this file in this PR? If it's not needed, please remove the whole root level html folder.

Reply:
The file existed from before. The whole html folder is generated for documentation purpose